doc/modules/ROOT/pages/index.adoc

```diff
@@ -107,15 +107,15 @@ $ java -Dserver.port=8080 -jar target/springboot-tomcat-session-replication-0.1.
 Open another terminal session and start the second application:
 [source,bash]
 ----
-java -Dserver.port=8081 -jar target/springboot-tomcat-session-replication-0.1.0.jar
+$ java -Dserver.port=8081 -jar target/springboot-tomcat-session-replication-0.1.0.jar
 ----
 
-Now, you can make calls to application endpoints to put data and get it back. Open another terminal session and send a request to `/put` endpoint. This will set the key value pair as a session attribute. Since session data is stored in Hazelcast distributed map, we can read this attribute from the other application instance using the same cookie:
+Now, you can make calls to application endpoints to put data and get it back. Open another terminal session and send a request to `/put` endpoint. This will set the key-value pair as a session attribute. Since session data is stored in Hazelcast distributed map, we can read this attribute from the other application instance using the same cookie:
```

Comment:
```suggestion
Now, you can issue HTTP calls to application endpoints to put data and get it back. Open another terminal session and send a request to `/put` endpoint. This will set the key-value pair as a session attribute. 

Since session data is stored in Hazelcast distributed map, we can read this attribute from the other application instance using the same cookie:
```

_make calls_ sounds like a suboptimal choice of words